strapi/strapi
# ML: Fix format of video duration

### What does it do?

It changes the way the length of a video is formatted. The duration has nothing to do with an internationalized date and can cause problems as described in the issue attached. Duration doesn't need to be localized I believe and is universal.

The utility isn't used anywhere else yet.

### Why is it needed?

To properly show the duration of a video.

### How to test it?

- Upload a video to the ML
- Check the displayed duration matches the real duration

### Related issue(s)/PR(s)

- Fixes https://github.com/strapi/strapi/issues/12868
